src/net/sf/mpxj/primavera/PrimaveraReader.java

```diff
@@ -692,33 +863,39 @@ private void addTaskUDFValue(Task task, Row row)
                break;
             }
 
+            case BOOLEAN:
+            {
+               value = STATICTYPE_UDF_MAP.get(row.getString("udf_text"));
```

Comment:
I understand that. However, being selfish for a second, my specific use case is hurt by lumping text fields and P6 indicator fields together. I need to be able to allow users to select a "text" field. Users did not like having tons of indicator fields mixed in containing "nonsense" values (and the indicator fields cut into the max text field count imposed by the core MPXJ MS Project model). My client code could maybe scan each text column for value that look like indicators and try to eliminate them, but that's definitely a kludge.

Any thoughts on how to make everyone happy?